DEPLOYNOTES.rst

```diff
@@ -3,6 +3,14 @@
 Deploy and Upgrade notes
 ========================
 
+0.13
+----
+
+* Member search requires an updated solrconfig.xml. Copy
+  `solr_conf/conf/solrconfig.xml` to your configured Solr configset
+  directory.
```

Comment:
This will likely also require a reload for the new configset (but as noted in Slack), not a reindex.

Reply:
Were you able to confirm that reload was sufficient? Happy to adjust the deploy notes